Sources/App/Models/Package.swift

```diff
@@ -86,6 +92,19 @@ extension Package: Hashable {
 }
 
 
+extension Package {
+    enum PlatformCompatibility: String, Codable {
+        case ios
+        case macos
+        // TODO: decide if we want to include macosArm here
+        case macosArm
```

Comment:
If we keep `macosArm` we need to change the update SQL both in this PR and in the post-merge SQL. They currently map any `macos-%` to `macos`.

Reply:
FWIW, I'm leaning towards keeping the SQL simpler and dropping it. It's already not an important platform discriminator and it's going to be even less so in the future.

Reply:
Certainly, for platform filters, we don't need it and it only makes the feature more complicated. I'd argue that at this stage, we should stop differentiating this across the whole system. If you're not compatible with *both* platforms, you're not compatible with macOS. Removing the whole feature is a discussion outside this PR, though.

Reply:
Yep, agreed! I'll update the PR now.